src/components/group/obs-group.component.tsx

```diff
@@ -33,7 +33,7 @@ export const ObsGroup: React.FC<FormFieldProps> = ({ question, onChange }) => {
       if (FieldComponent) {
         return (
           <div className={classNames(styles.flexColumn, styles.obsGroupColumn)} key={keyId}>
-            <div className={styles.parentResizer}>
+            <div className={styles.obsGroupButton}>
```

Comment:
@gitcliff is this class name targeting a button group? a button? or it's just a container wrapper?

Reply:
@samuelmale its targeting the button

Reply:
Which button are you targeting?

Reply:
each button in the obs group

Reply:
@gitcliff If this is targeting a button and not a container, can you make the underlying CSS selector(s) more specific to buttons? (It's currently ambiguous)